packages/angular/pwa/pwa/index.ts

```diff
@@ -117,7 +117,7 @@ export default function(options: PwaOptions): Rule {
     }
 
     // Add manifest to asset configuration
-    const assetEntry = join(normalize(project.root), 'src', 'manifest.webmanifest');
+    const assetEntry = join(project.root, project.sourceRoot ?? 'src', 'manifest.webmanifest');
```

Comment:
Use posix.join to retain the same behaviour on non posix file systems.

Reply:
Updated.